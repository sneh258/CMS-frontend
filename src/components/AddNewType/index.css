.add-new-type{
    border: 2px solid rgb(87, 5, 168);
    border-radius: 15px 15px;
    margin-top: 20px;
    margin-left: 15px;
    margin-right: 15px;
    display: flex;
    background-color:rgb(87, 5, 168) ;
    color: white;
    justify-content: center;
}