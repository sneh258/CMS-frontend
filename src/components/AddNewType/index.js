import React from 'react';
import './index.css';

export default function AddNewType() {
  return <div className='add-new-type'><p>Add new Type</p></div>;
}
