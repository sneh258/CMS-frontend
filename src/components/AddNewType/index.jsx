import React from 'react';
import PropTypes from 'prop-types';
import './index.css';

export default function AddNewType({data}) {
  return <div className='add-new-type'><p>{data}</p></div>;
}

AddNewType.propTypes = {
  data: PropTypes.array.isRequired,
};