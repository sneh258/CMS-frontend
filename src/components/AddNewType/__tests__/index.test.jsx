import React from 'react';
import AddNewType from '..';
import { render } from '@testing-library/react';

const mockNavigate = jest.fn();
jest.mock('react-router-dom', () => ({
  useNavigate: () => mockNavigate,
}));

describe('Add new Content Type', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<AddNewType data="value" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
