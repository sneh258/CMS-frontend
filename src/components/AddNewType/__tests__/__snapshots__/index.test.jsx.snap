// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add new Content Type should render correctly 1`] = `
<DocumentFragment>
  <div
    class="add-new-type"
  >
    <p>
      value
    </p>
  </div>
</DocumentFragment>
`;
